RELEASING.md

```diff
@@ -49,7 +36,7 @@ green, you may create a release as follows:
    [google-cloud](https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud)
    and
    [stackdriver](https://github.com/googleapis/google-cloud-ruby/blob/master/stackdriver)
-   last, in case of dependency changes. (See steps 15 and 16, below.)
+   last, in case of dependency changes. (See steps 13 and 14, below.)
 
```

Comment:
I think we should insert a step here to `git pull`, since this is inside the repeated loop, and when releasing multiple gems master may have been updated since the `git pull` in step 1. above.